src/components/home/WhoAreWe.tsx

```diff
@@ -1,5 +1,43 @@
+import Image from "next/image";
+import groupHome from "@/public/home/groupHome.webp";
+import Button from "@/components/Button";
+
 const WhoAreWe = () => {
-  return <div>Who Are We?</div>;
+  return (
+    <div className="relative h-[800px] w-full">
```

Comment:
```suggestion
    <div className="relative h-96 w-full">
```

avoid using px where possible